grid/social-media/upload-url/profile.supr

```diff
@@ -0,0 +1,222 @@
+
+"""
+Upload Assets via URL to Social Media sites
+Upload images and videos available via an URL, onto social media such as Instagram, Facebook, or Twitter.
+"""
+
+name = "social-media/upload-url"
```

Comment:
Nit: consider naming the profile like "upload-by-url" or "upload-from-url", we are not uploading the URL itself. ;)